apps/ledger-live-desktop/src/renderer/families/celo/SimpleOperationFlowModal/Body.js

```diff
@@ -0,0 +1,223 @@
+// @flow
+import invariant from "invariant";
+import React, { useState, useCallback } from "react";
+import { compose } from "redux";
+import { connect, useDispatch } from "react-redux";
+import { Trans, withTranslation } from "react-i18next";
+import { createStructuredSelector } from "reselect";
+import { SyncSkipUnderPriority } from "@ledgerhq/live-common/lib/bridge/react";
+import Track from "~/renderer/analytics/Track";
+
+import { UserRefusedOnDevice } from "@ledgerhq/errors";
+
+import { getAccountBridge } from "@ledgerhq/live-common/lib/bridge";
+import useBridgeTransaction from "@ledgerhq/live-common/lib/bridge/useBridgeTransaction";
+
+import type { StepId, StepProps, St, Mode } from "./types";
+import type { Account, Operation } from "@ledgerhq/live-common/lib/types";
+import type { TFunction } from "react-i18next";
+import type { Device } from "@ledgerhq/live-common/lib/hw/actions/types";
+
+import { addPendingOperation } from "@ledgerhq/live-common/lib/account";
+import { updateAccountWithUpdater } from "~/renderer/actions/accounts";
+
+import { getCurrentDevice } from "~/renderer/reducers/devices";
+import { closeModal, openModal } from "~/renderer/actions/modals";
+
+import Stepper from "~/renderer/components/Stepper";
+import StepInfo, { StepInfoFooter } from "./steps/StepInfo";
+import GenericStepConnectDevice from "~/renderer/modals/Send/steps/GenericStepConnectDevice";
+import StepConfirmation, { StepConfirmationFooter } from "./steps/StepConfirmation";
+import logger from "~/logger/logger";
+
+type OwnProps = {|
+  stepId: StepId,
+  onClose: () => void,
+  onChangeStepId: StepId => void,
+  params: {
+    account: Account,
+    parentAccount: ?Account,
+  },
+  name: string,
+  mode: Mode,
+|};
+
+type StateProps = {|
+  t: TFunction,
+  device: ?Device,
+  accounts: Account[],
+  device: ?Device,
+  closeModal: string => void,
+  openModal: string => void,
+|};
+
+type Props = OwnProps & StateProps;
+
+const steps: Array<St> = [
+  {
+    id: "info",
+    label: <Trans i18nKey="celo.simpleOperation.steps.info.title" />,
+    component: StepInfo,
+    footer: StepInfoFooter,
+  },
+  {
+    id: "connectDevice",
+    label: <Trans i18nKey="celo.simpleOperation.steps.connectDevice.title" />,
+    component: GenericStepConnectDevice,
+    onBack: ({ transitionTo }: StepProps) => transitionTo("info"),
+  },
+  {
+    id: "confirmation",
+    label: <Trans i18nKey="celo.simpleOperation.steps.confirmation.title" />,
+    component: StepConfirmation,
+    footer: StepConfirmationFooter,
+  },
+];
+
+// returns the first error
```

Comment:
Why just the first error? If there's multiple, wouldn't we want to show users all the errors?

Reply:
Yes, we do show only the first error, that's the case in all modals. Usually there's only one error anyway, but I agree it's a bit weird